docs/reference/ml/df-analytics/apis/get-trained-models.asciidoc

```diff
@@ -113,6 +113,10 @@ Information on the creator of the trained model.
 (<<time-units,time units>>)
 The time when the trained model was created.
 
+`description`:::
```

Comment:
Nit: we order the properties alphabetically, so `description` comes after `default_field_map`. However, it is not an important, but rather an annoying suggestion, so feel free to merge the PR, and I'll amend the docs later.